src/assets/drizzle/styles/components/swatch.css

```diff
@@ -0,0 +1,19 @@
+:root {
+  --Swatch-width: 10em;
+  --Swatch-height: calc(var(--Swatch-width) * (9 / 16)); /* 16:9 */
```

Comment:
You missed nothing, I changed up the variable and added the comment based on [this comment](https://github.com/cloudfour/drizzle/pull/45#discussion_r62238091). 🙂